bin/pygrb/pycbc_pygrb_plot_snr_timeseries

```diff
@@ -47,46 +48,45 @@ __program__ = "pycbc_pygrb_plot_snr_timeseries"
 # =============================================================================
 # Functions
 # =============================================================================
-# Function to load time and necessary SNR data from a trigger/injection file
-def load_data(input_file, vetoes, opts, injections=False, sim_table=False):
-    """Build a dictionary containing time and SNR data extracted from
-    a trigger/injection file"""
+# Load trigger data
+def load_data(input_file, vetoes, ifos, opts):
+    """Load data from a trigger/injection file"""
+    logging.info("Loading triggers...")
 
-    # Inizialize the dictionary
+    # Initialize the dictionary
     data = {}
     data['time'] = None
-    snr_type = opts.y_variable
-    data[snr_type] = None
+    data['coherent'] = None
+    data['null'] = None
+    data['reweighted'] = None
+    data['single'] = dict((ifo, None) for ifo in ifos)
```

Comment:
No dictionary needed here with hdf5.  For now, as we wait for the injections files to settle, simply have this function do
```
if injections:
    trigs_or_injs = ppu.load_triggers(input_file, vetoes)
else:
    trigs_or_injs = ppu.load_triggers(input_file, vetoes)

return trigs_or_injs
```
Later on, the first one will become `load_injections(...)`.

Then in the main part of the script, you want to build the correct keys for the x and y quantities.  E.g.,
```
y_key = opts.ifo+'/'+'snr_'+opts.ifo.lower() if snr_type == 'single' else 'network/'+snr_type+'_snr'
```
should work.  Then you need to do something similar for the time (`x_key`), manipulate the times, and finally just
```
plu.pygrb_plotter([trig_data[x_key], trig_data[y_key]],
                  [inj_data[x_key], inj_data[y_key]],
                  "Time since %.3f (s)" % (central_time), y_label,
                  opts, cmd=' '.join(sys.argv))
```